tools/clang/test/HLSLFileCheck/hlsl/control_flow/attributes/unroll/partial_cond.hlsl

```diff
@@ -3,7 +3,6 @@
 // CHECK: call float @dx.op.dot3
 // CHECK: call float @dx.op.dot3
 // CHECK: call float @dx.op.dot3
-// CHECK: call float @dx.op.dot3
```

Comment:
Instead of removing this, we should replace this check with a check for whatever `dot3` got simplified into. Ditto for other places.